topsy.el

```diff
@@ -104,6 +114,22 @@ Return non-nil if the minor mode is enabled."
 
 ;;;; Functions
 
+(defun topsy--header-string ()
+  "Return string found by `topsy-fn' or line above window start."
+  (or (when-let ((header (and topsy-fn (funcall topsy-fn))))
```

Comment:
I wonder if we should set `topsy-fn` to `#'ignore` by default, then we could omit the condition.  What do you think?